### PR TITLE
feat: Create GET About Page 

### PR DESCRIPTION
**Pull request by nverbic**
Closes: https://github.com/Women-Coding-Community/wcc-backend/issues/115

Add GET API for About Us page.

// TODO: Use Page or HeroSection record?
For now use Page. Page is going to be renamed to a more general component and replace HeroSection and Section components as well.

// TODO: Add id to the below record declaration?
When the DB is setup, id will be added to the specific  page declarations (like AboutUs page, Code of conduct page etc)

Screenshot (125)

Screenshot (126)

### CODE REVIEW

File: src/main/java/com/wcc/platform/domain/cms/pages/AboutUs.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.wcc.platform.domain.cms.pages;
+
+import com.wcc.platform.domain.cms.attributes.Contact;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import java.util.List;
+
+// TODO: Get photo for About Us hero section.
```
Comment (dricazenck) [inline src/main/java/com/wcc/platform/domain/cms/pages/AboutUs.java:8]: Take from Figma, you can upload to Google drive and we can use the link for download, similar to the landing page test.

File: src/main/java/com/wcc/platform/domain/cms/PageType.java [hunk 2]
```diff
@@ -15,7 +15,8 @@ public enum PageType {
   MENTORSHIP("mentorshipPage.json"),
   TEAM("teamPage.json"),
   PROG_BOOK_CLUB("bookClubPage.json"),
-  EVENT_FILTERS("eventsFiltersSection.json");
+  EVENT_FILTERS("eventsFiltersSection.json"),
+  ABOUT_US("aboutUs.json");
```
Comment (dricazenck) [inline src/main/java/com/wcc/platform/domain/cms/PageType.java:19]:
```suggestion
  ABOUT_US("aboutUsPage.json");
```

File: src/main/java/com/wcc/platform/domain/cms/pages/AboutUs.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+// TODO: Use Page or HeroSection record?
+// TODO: Add id to the below record declaration?
+// TODO: Add description after title to Contact record (to be able to add the description from the
```
Comment (dricazenck) [inline src/main/java/com/wcc/platform/domain/cms/pages/AboutUs.java:11]: You can add description into Contact object, as non mandatory attribute.

File: src/main/java/com/wcc/platform/domain/cms/pages/AboutUs.java [hunk 4]
```diff
@@ -0,0 +1,14 @@
+package com.wcc.platform.domain.cms.pages;
+
+import com.wcc.platform.domain.cms.attributes.Contact;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import java.util.List;
+
+// TODO: Get photo for About Us hero section.
+// TODO: Use Page or HeroSection record?
+// TODO: Add id to the below record declaration?
```
Comment (dricazenck) [inline src/main/java/com/wcc/platform/domain/cms/pages/AboutUs.java:10]: Please add as string for now

File: src/main/java/com/wcc/platform/domain/cms/pages/AboutUs.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+// TODO: Add description after title to Contact record (to be able to add the description from the
+// Figma About Us contact example)?
+public record AboutUs(
```
Comment (dricazenck) [inline src/main/java/com/wcc/platform/domain/cms/pages/AboutUs.java:13]:
```suggestion
public record AboutUsPage(
```

File: src/main/java/com/wcc/platform/domain/cms/pages/AboutUs.java (hunk 4 above)
Comment (nverbic) [reply src/main/java/com/wcc/platform/domain/cms/pages/AboutUs.java]: As agreed during the meeting, we will add ids to all pages in another PR.

**Review by dricazenck — APPROVED** [review]

**Pull request merged**